libsolidity/ast/Types.h

```diff
@@ -1,18 +1,18 @@
 /*
-    This file is part of solidity.
+	This file is part of solidity.
```

Comment:
I just blanket-converted all spaces to tabs in this file, did we also want the comment to use tabs?

Reply:
This is a very old file, please just keep existing lines in comments.